src/researchhub_document/views/researchhub_unified_document_views.py

```diff
@@ -117,6 +140,7 @@ def update(self, request, *args, **kwargs):
         return update_response
 
     def _get_document_filtering(self, query_params):
+        use_v2_hot_score = query_params.get('hot_v2') == 'true'
```

Comment:
```suggestion
        use_v2_hot_score = query_params.get('hot_v2', False)
```

Reply:
@le-sun I need here explicit check of 'true' string. I want to avoid the FE potentially adding the query var with no val yet still resulting in v2 feed results